qutebrowser/browser/webengine/interceptor.py

```diff
@@ -71,6 +115,10 @@ def interceptRequest(self, info):
 
         url = info.requestUrl()
         first_party = info.firstPartyUrl()
+        # Per QWebEngineUrlRequestInfo::ResourceType documentation, if we fail
+        # our lookup, we should not fall back to ResourceTypeUnknown
```

Comment:
I think you're parsing those docs wrong. They say:

> Note: For forward compatibility all values not matched should be treated as unknown, not just ResourceTypeUnknown.

Which I parse as "if you get another value, treat it as `ResourceTypeUnknown`, don't assume `ResourceTypeUnknown` is the only valid unknown value."

Reply:
ohh, I see, I'll probably modify this case to default to ResourceTypeUnknown then :)